_data/changelogs/component-file-input.yml

```
@@ -2,6 +2,40 @@ title: File input
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Improved the file input experience for voice command and screen reader users.
+    summaryAdditional: Voice command users can now interact with the component by speaking the visible instructions text.
+      Additionally, screen reader users now have access to both the instructions text
+      and the file selection status.
+    affectsAccessibility: true
+    affectsJavascript: true
+    affectsStyles: true
+    githubPr: 5213
+    githubRepo: uswds
+    versionUswds: 3.5.0
+  - date: NNNN-NN-NN
+    summary: Improved consistency and visibility of disabled styles.
+    summaryAdditional: Form elements with the `disabled` or `aria-disabled` attribute now get consistent styling and have proper color contrast.
```

Part of this change removed the disabled variant `.json` file, which made it necessary to remove the `file-input-disabled` from the `file-input` partial to resolve build errors. 

@amyleadem mentioned it might be a good idea to update this changelog item to reflect this as well!

```suggestion
    summaryAdditional: Form elements with the `disabled` or `aria-disabled` attribute now get consistent styling and have proper color contrast. Removed `file-input-disabled` preview.
```

Once we come up with our disabled forms guidance we can consider adding the disabled preview back in.

---

@mahoneycm Would you be able to add the changelog entry directly to https://github.com/uswds/uswds-site/pull/2114? I think crediting the removal of the disabled variant to #2114 makes sense to me the more I think about it.

---

Yeah! I'll do that now!